config/initializers/test_track_generate_timestamp.rb

```diff
@@ -0,0 +1,9 @@
+BUILD_TIMESTAMP_FILE_PATH = 'testtrack/build_timestamp'.freeze
+
+if Rails.env.test? || Rails.env.development?
+  TestTrack::BuildTimestamp = Time.zone.now.iso8601
+elsif File.exist?(BUILD_TIMESTAMP_FILE_PATH) && File.readable?(BUILD_TIMESTAMP_FILE_PATH) && !File.zero?(BUILD_TIMESTAMP_FILE_PATH)
+  TestTrack::BuildTimestamp = File.read(BUILD_TIMESTAMP_FILE_PATH)
+else
+  raise Exception.new "Failed to read timestamp"
```

Comment:
This could use more context. If the app fails to startup we should provide plenty of details on why and how to fix.

Something like: “TestTrack failed to load the required build timestamp. Ensure testtrack-cli build-timestamp is run and the timestamp file is present”

Something like that but better 😁

Reply:
Any reason we're raising a bare exception instead of just the text, which would result in a StandardError?